easybuild/easyconfigs/t/Tk/Tk-8.6.4-intel-2016a-no-X11.eb

```diff
@@ -20,6 +20,7 @@ dependencies = [
     ('zlib', '1.2.8'),
 ]
 
+# To be clear: this will still require the X11 to be present.
```

Comment:
if you add this, include a pointer to https://github.com/hpcugent/easybuild-easyconfigs/issues/2261